src/randgraphs.jl

```diff
@@ -96,11 +96,10 @@ end
 
 function _suitable(edges::Set{Edge}, potential_edges::Dict{Int, Int})
     isempty(potential_edges) && return true
-    for (s1, s2) in combinations(collect(keys(potential_edges)), 2)
-        if (s1 > s2)
-            s1, s2 = s2, s1
-        end
-        ∉(Edge(s1, s2), edges) && return true
+    list = keys(potential_edges)
+    for s1 in list, s2 in list
+        s1 >= s2 && continue
+        (Edge(s1, s2) ∉ edges) && return true
```

Comment:
I don't think there would be a sensible advantage: n^2 -> n choose 2 is just a factor 2 in the number of loops, and continuing is cheap. I would apply your comment if I could avoid branching in the loop, but since list is not ordered I would to add an extra 
```julia
 if (s1 > s2)
    s1, s2 = s2, s1 
end
```
so, I prefer to leave it as it is